src/crypto/BUILD.gn

```diff
@@ -20,6 +20,8 @@ import("${chip_root}/build/chip/buildconfig_header.gni")
 
 import("crypto.gni")
 
+import("${chip_root}/src/lib/lib.gni")
```

Comment:
this adds a dependency from crypto to lib. This seems off. Why not a crypto gni if these are crypto related configuration options?